contrib/QtCreator/MEGAtests/MEGAtest_unit/MEGAtest_unit.pro

```diff
@@ -16,15 +17,30 @@ CONFIG += USE_MEDIAINFO
 CONFIG += USE_LIBRAW
 CONFIG += USE_FFMPEG
 CONFIG -= qt
+CONFIG += object_parallel_to_source
```

Comment:
We need this so that the QMake build doesn't break. By default, QMake puts all object files into a flat directory which means that if you have utils.cpp under src and utils.cpp under test/unit your build breaks because the object files overwrite each other.